docs/security/threat-model.md

```diff
@@ -0,0 +1,219 @@
+We can create a configuration to reject by default requests where the API
+version not cover by the policy.  We should warning policies developers to cover
+all the supported API version in theirs tests and reject all of others.
+
+
+## Threat ID 15 - Attacker deploys privileged container to node running  Webhook controller
+An attacker, who has rights to deploy privileged containers to the cluster, creates
+a privileged  container on the cluster node where the admission controller webhook operates.
+
+**Mitigation**
+Admission controller uses restrictive policies to prevent privileged  workloads
+
+**To do**
+I do not know if Kuberwarden can help on this. Kubewarden does not have access to
+containers running in the cluster node.
+
+
+## Threat ID 16 - Attacker mounts a privileged node hostpath allowing  modification of Webhook controller configuration
+An attacker, who has rights to deploy hostPath volumes with workloads, creates a
+volume  which allows for access to the admission controller pod’s files.
+
+**Mitigation**
+Admission controller uses restrictive policies to prevent privileged  workloads
+
+**To do**
+We can add a recommended policy in the `kubewarden-default` Helm chart to prevent this.
+
+
+## Threat ID 17 - Attacker has privileged SSH access to cluster node  running admission webhook
+An attacker is able to log into cluster nodes as a privileged user via SSH.
+**Mitigation**
+N/A
+
+I don't think Kubewarden can help on this
+
+
+## Threat ID 18 - Attacker uses policies to send confidential data from  admission requests to external systems
+An attacker is able to configure a policy that listens to admission requests and
+sends  sensitive data to an external system.
+
+**Mitigation**
+Strictly control external access for webhook
+
+Kubewarden policies run in a restrictive environment. They do not have network access.
+
+
+## Threat Kubewarden ID 1 - Bootstrapping of trust for admission controller
+Assuming a trusted but fresh Kubernetes cluster, an attacker is able to compromise the Kubewarden stack before any of the policies securing it is deployed and enforcing. For example, by using unsigned and malicious images for kubewarden-controller, policy-server, or any of the Kubewarden dependencies (cert-manager) or optional dependencies (grafana, prometheus..), or by compromising the Helm charts payload.
+
+** Mitigation **
+1. Kubewarden provides a Software Bill Of Materials, which lists all images needed. This aids with Zero-Trust.
+  The Kubernetes Administrator must verify the Kubewarden images (and its dependencies' images and charts) out of the Kubernetes cluster, in a trusted environment. This can be done with `cosign`, for example.
+  Incidentally, this is part of the implementation needed for air-gapped installations.
+ 2. Use signed Helm charts, and verified digests (instead of tags) for Kubewarden images in those Helm charts. This doesn't secure dependencies though.
```

Comment:
```suggestion
 2. Use signed Helm charts, and verified digests instead of tags for Kubewarden images in those Helm charts. 
 This doesn't secure dependencies though.
```